.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: c
 cache: ccache
+dist: trusty
```

Comment:
Well, it says "unlikely this year", but it is dated last year. So that it has to mean that we ought to be entitled to expect it now, because it's not that year anymore :-)

Some background. As mentioned in Android request, sizable portion of additional CPU-hour[!] and resources went to unpacking Android NDK, which is/has to be done on per-test basis. And pulling gcc-5 and clang-3.6 is not exactly gratis either and 1/2 of it *is* in fact wasted as is. I mean since it's done on per-test basis, clang pull is wasted in gcc tests and vice versa. Last optimization with conditional install of *some* packages reduced CPU time by ~45 minutes, i.e. there is a *lot* to spare on unused prerequisite installations.